libsabergen/sabergen.py

```diff
@@ -75,22 +75,19 @@ def convert_to_ogg(self, song_path):
 
             # Command will generate a new file with the same name, except .ogg
             # -c:a codec, -y overwrite if output exists, -q quality specifier
-            subprocess.run("ffmpeg -y -i {} -c:a libvorbis -q:a 8 {}".format(song_path, ogg_path),
-                           shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+            result = subprocess.run(
+                "ffmpeg -y -i {} -c:a libvorbis -q:a 8 {}".format(song_path, ogg_path),
+                shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
 
-            if self.debug:
-                # TODO: How best to return stdout/stderr from a library?
-                pass
+            # ffmpeg writes most of its normal output to stderr
+            logging.info(result.stdout)
+            logging.info(result.stderr)
```

Comment:
Did you come across an instance were standard error was written to when the return code was 0?

Reply:
Nevermind, I can't read.